api/src/main/java/org/pmiops/workbench/api/CohortsController.java

```diff
@@ -133,6 +141,7 @@ public Cohort apply(org.pmiops.workbench.db.model.Cohort cohort) {
     try {
       // TODO Make this a pre-check within a transaction?
       dbCohort = cohortDao.save(dbCohort);
+      userRecentResourceService.updateCohortEntry(workspace.getWorkspaceId(), userProvider.get().getUserId(), dbCohort.getCohortId(), now);
```

Comment:
This is the most logical place to update the cohort entry for a clone/save cohort event.